lib/hashauth.js

```diff
@@ -74,16 +74,18 @@ hashauth.init = function init(client, $) {
     }
     return false;
   };
-  
+
   hashauth.requestAuthentication = function requestAuthentication (eventOrNext) {
     var translate = client.translate;
     hashauth.injectHtml();
     $( '#requestauthenticationdialog' ).dialog({
-        width: 350
-      , height: 240
+        width: 400
+      , height: 270
+      , closeText: ''
       , buttons: [
         {
-          text: translate('Update')
+          id: 'requestauthenticationdialog-btn'
```

Comment:
Search is better by ID that `$(this).parent().find('button.ui-button-text-only')`